jarviscli/plugins/international_day.py

```diff
@@ -0,0 +1,128 @@
+import csv
+import datetime
+from colorama import Fore
+from plugin import plugin
+
+
+@plugin("international day")
+class InternationalDay:
+    """
+    Jarvis plugin that prints the
+    International Day of the current date or of the
+    input date the user enters.
+    """
+    def __call__(self, jarvis, s):
+        """
+        Calls the needed methods to execute the plugin.
+        """
+        jarvis.say("\nWhich date would you like to know"
+                   " there is an international day for? ")
+        # Formats the printed message
+        message = Fore.RESET + Fore.YELLOW + \
+            "[1]" + Fore.RESET + " For today's!\n"
+        message += Fore.RESET + Fore.YELLOW + "[2]" + Fore.RESET + \
+            " For another day...\n"
+        jarvis.say(message)
+        # Takes the user's input choice.
+        choice = jarvis.input("Please enter your choice: ")
+        # Loop to validate the input choice.
+        while True:
+            try:
+                # Find International Day for the current date.
+                if choice == "1":
+                    current_date = self.find_date()
+                    current_day, current_month = self.split_date(current_date)
+                    # Open the csv file that contains the International Days
+                    # which is located in the data folder.
+                    with open(r'jarviscli\data\international_days.csv')\
```

Comment:
\ doesn't work cross-platform (Linux and I guess MacOS?). Either use / - which does also work on Windows. Or os.path.join('jarviscli', 'data', 'international_days.csv'). Or os.sep, which gives you '/' or '\' depending on your platform.